examples/low_frequency/general/twin_builder/lti_rom_sml.py

```diff
@@ -0,0 +1,252 @@
+input_dir = downloads.download_twin_builder_data(
+    training_data_folder, True, temp_folder.name
+)
+
+# Download data from example_data repository
+
+data_folder = os.path.join(input_dir, "LTI_training")
+
+# Unzip training data and parse ports names
+
+downloads.unzip(os.path.join(input_dir, training_data_folder), data_folder)
+ports_names_file = "Input_PortNames.txt"
+
+
+# ## Get ports information from file
+
+
+def get_ports_info(ports_file):
+    with open(ports_file, "r") as PortNameFile:
+        My_s = []
+        line = PortNameFile.readline()
+        line_list = list(line.split())
+        for i in range(len(line_list)):
+            My_s.append("Input" + str(i + 1) + "_" + line_list[i])
+
+        line = PortNameFile.readline()
+        line_list = list(line.split())
+        for i in range(len(line_list)):
+            My_s.append("Output" + str(i + 1) + "_" + line_list[i])
+    return My_s
+
+
+My_s = get_ports_info(os.path.join(data_folder, ports_names_file))
+
+# ## Launch Twin Builder
+#
+# Launch Twin Builder using an implicit declaration and add a new design with
+# the default setup.
+
+project_name = os.path.join(temp_folder.name, "LTI_ROM.aedt")
+tb = TwinBuilder(
+    project=project_name, version=AEDT_VERSION, non_graphical=NG_MODE, new_desktop=True
+)
+
+# ## Build the LTI ROM with specified configuration file
+
+install_dir = tb.odesktop.GetRegistryString("Desktop/InstallationDirectory")
+fitting_exe = os.path.join(install_dir, "FittingTool.exe")
+path = '"' + fitting_exe + '"' + "  " + '"t"' + "  " + '"' + data_folder + '"'
+process = subprocess.Popen(
+    path, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+)
+tb.logger.info("Fitting the LTI ROM training data")
+exec = True
+startTime = datetime.datetime.now()
+execTime = 0.0
+while (
+    exec and execTime < 60.0
+):  # limiting the fitting process execution time to 1 minute
+    out, err = process.communicate()
+    execTime = (datetime.datetime.now() - startTime).total_seconds()
+    if "An LTI ROM has been generated" in str(out):
+        process.terminate()
+        exec = False
+
+rom_file = ""
+model_name_sml = ""
+
+for i in os.listdir(data_folder):
+    if i.endswith(".sml"):
+        model_name_sml = i.split(".")[0]
+        rom_file = os.path.join(data_folder, i)
+
+if os.path.exists(rom_file):
+    tb.logger.info("Built intermediate ROM file successfully at: %s", rom_file)
+else:
+    tb.logger.info("ROM file does not exist at the expected location : %s", rom_file)
+
+
+# ## Import the ROM component model
+
+is_created = tb.modeler.schematic.create_component_from_sml(
+    input_file=rom_file, model=model_name_sml, pins_names=My_s
+)
+os.remove(rom_file)
+tb.logger.info("LTI ROM model successfully imported.")
+
+# ## Import the ROM component model in Twin Builder
+#
+# Place components to create a schematic.
+
+# Define the grid distance for ease in calculations
+
+G = 0.00254
```

Comment:
Can you use another variable name ? I like that `G` is specified as a constant but the name isn't clear to me.